presto-orc/src/main/java/io/prestosql/orc/StripeReader.java

```diff
@@ -332,12 +333,13 @@ private InputStreamSources createDictionaryStreamSources(Map<StreamId, Stream> s
             ColumnMetadata<ColumnEncoding> encodings)
             throws InvalidCheckpointException
     {
+        checkState(rowsInRowGroup.isPresent(), "Cannot create row groups if row group info is missing");
```

Comment:
Maybe do this instead:
```java
int rowsInRowGroup = this.rowsInRowGroup.orElseThrow(() -> new IllegalStateException("Cannot create row groups if row group info is missing"))
```
Then the code below doesn't need to change.

Reply:
Done